pyclustree/__init__.py

```diff
@@ -2,8 +2,6 @@
 
 from ._clustree import clustree
 
-pyclustree = clustree
-
-__all__ = ["clustree", "pyclustree"]
+__all__ = ["clustree"]
```

Comment:
I wanted to also make it available as pyclustree in case people mistype because the differentiation between pyclustree the package and clustree the package name might not be very clear. What do you think?

Reply:
For me it's ok to keep both function names. The only ugly thing is the empty section in the API docs (https://pyclustree.readthedocs.io/latest/autoapi/pyclustree/index.html#pyclustree.pyclustree) :)